docs/fundamentals/modernize-dev-workflow.md

```diff
@@ -0,0 +1,216 @@
+# Modernizing Your Dev Workflow
+
+Using Nx, you can add Cypress, Jest, Prettier, and Nest into your dev workflow.
+
+Of course, it's not the case that Cypress is always better than Protractor or Nest is always better than say Express. There are tradeoffs. But in many situations, for many projects, these innovative tools offer a lot of advantages.
+
+Adding these tools to the dev workflow is challenging in a regular CLI project. The choice you have is not between Protractor or Cypress, but between a hacked-up setup for Cypress and a great CLI setup for Protractor. Nx changes that!
+
+## Nest
+
+<img src="./nest-logo.png">
+
+Nest is a Node.js framework designed for building scalable server-side applications. In many ways, Nest is familiar to Angular developers:
+
+- It has excellent TypeScript support.
+- Its dependency injection system is similar to the one in Angular.
+- It emphasises testability.
+- Its configuration APIs are similar to Angular as well.
+
+Many conventions and best practices used in Angular applications can be also be used when using Nest.
+
+To create a new Nest application, run:
```

Comment:
I feel like this content is a repeat of content in the full-stack development guide